src/pybind/mgr/diskprediction_cloud/common/__init__.py

```diff
@@ -29,7 +29,7 @@ class TimeoutError(Exception):
     pass
 
 
-def timeout(seconds=10, error_message=os.strerror(errno.ETIME)):
+def timeout(seconds=10, error_message=os.strerror(errno.ETIMEDOUT)):

```

Comment:
i think ETIME is also defined by POSIX. see https://pubs.opengroup.org/onlinepubs/9699919799/functions/ioctl.html . FWIW, IEEE Std 1003.1-2017 is also known as POSIX.1-2017. i think it's just FreeBSD does not care to support XSI STREAMS (properly).

please fix the commit message.

Reply:
@tchaikov 
You are correct.  Most of the FreeBSD code creates a compatibility shim like:
```
/usr/local/include/libsync.h:39:#define ETIME ETIMEDOUT
```
Will amend